### DISCUSSION TITLE
Any way to see prior searches (history)?

### DISCUSSION THREAD

**Discussion by kperath** [ibhagwan/fzf-lua#1524] [Q&A] [answered]
With telescope, I could cycle back to prior search results and used the plugin https://github.com/nvim-telescope/telescope-smart-history.nvim which would memorize prior search results I made for specific pickers. So if I live_grep and I forget to resume from my last result, then I don't have to type out the word again and I can just press the up arrow to get my last search result or last few searches.

I know with grep you can use vims command history to see what you last searched. But this would be nice for live_grep, files and maybe in general as well.

Part of telescope config, if it helps better convey what I'm saying:
```lua
			require("telescope").setup({
				defaults = {
					layout_strategy = "vertical",
					layout_config = {
						vertical = { width = 0.5 },
					},
					mappings = { -- extend mappings
						i = {
							["<C-S>"] = actions.file_split,
							["<Down>"] = actions.cycle_history_next,
							["<Up>"] = actions.cycle_history_prev,
							["<c-enter>"] = actions.to_fuzzy_refine,
						},
					},
					history = {
						path = "~/.local/share/nvim/databases/telescope_history.sqlite3",
						limit = 100,
					},
				},

```

**Comment by ibhagwan** [comment, marked as answer]
https://github.com/ibhagwan/fzf-lua/wiki#how-do-i-setup-input-history-keybinds

**Reply by kperath** [reply]
Amazing! Classic RTFM moment 🤦‍♂️ Thank you for the quick reply 🙇‍♂️

**Reply by ibhagwan** [reply]
You’re welcome 😄
